geometry_msgs/PointStamped pose
---
bool result
